### PR TITLE
Feature: Added banner pre-title

### PR DESCRIPTION
**Pull request by bspeare**
# How to test

- Go to https://uids.brand.uiowa.edu/branches/feature_banner_pre_title/components/detail/banner--default.html and click through all the banners and verify that a pre-title field is appearing above the banner title field in the same headline style as the banner title. 
  - The "Highlight" styles will render without the highlight background
- Review banner template changes
- Review `banner.scss` changes

### CODE REVIEW

**Comment by pyrello** [comment]
Is this supposed to be using the highlighted headline style for the pre-title? https://uids.brand.uiowa.edu/branches/feature_banner_pre_title/components/detail/banner--large_centered_left_banner.html

**Comment by bspeare** [comment]
> 

No, the "Highlight" styles will render without the highlight background` otherwise it gets pretty busy.

File: src/components/banner/banner.scss [hunk 1]
```diff
@@ -267,53 +340,29 @@
...
 
 // Set large heading size
-.headline--large.headline--serif,
-.headline--large {
-  font-size: 4.7rem;
-  font-size: containerClamp(2.5rem, 4.7rem);
-}
+:where(h2, h3, h4, h5, h6) {
+  &.headline--large.headline--serif,
+  &.headline--large {
```
Comment (pyrello) [inline src/components/banner/banner.scss:348]: I'm wondering why both of these lines are needed?

Comment (bspeare) [reply src/components/banner/banner.scss:348]: Good question, it is to override the modification to the sizes set in https://github.com/uiowa/uids/blob/main/src/components/typography/headings/headings.scss to make Zilla render as a comparable size to Roboto.

Comment (bspeare) [reply src/components/banner/banner.scss:346]: I removed one set that wasn't needed and added some supporting comments.  Thanks for pointing it out!

**Review by pyrello — APPROVED** [review]
Looks good.

**Pull request merged**